src/hotspot/share/classfile/klassFactory.cpp

```diff
@@ -155,7 +155,7 @@ static ClassFileStream* check_class_file_load_hook(ClassFileStream* stream,
       // Set new class file stream using JVMTI agent modified class file data.
       stream = new ClassFileStream(ptr,
                                    pointer_delta_as_int(end_ptr, ptr),
-                                   stream->source());
+                                   stream->source(), false, /* from_class_file_load_hook */ true);
```

Comment:
Can you comment what the `false` means as well please - looks odd to only document one of the boolean parameters.